shared/src/store/action-reducers/emergency-operation-center.ts

```diff
@@ -15,6 +39,25 @@ export class AddLogEntryAction implements Action {
     public readonly message!: string;
 }
 
+export class SendAlarmGroupAction implements Action {
```

Comment:
This class would be more more readable with an empty line between two properties.

Reply:
Done in 9fae3e2535a1c52601085fc6126acc7d4efa034b